### PR TITLE
WT-4989 Migrating Jenkins unit-test-long job to Evergreen

### PR DESCRIPTION
**Pull request by hbokhari**

### CODE REVIEW

File: test/evergreen.yml [hunk 1]
```diff
@@ -1027,6 +1027,16 @@ tasks:
       - func: "fetch artifacts"
       - func: "unit test"
 
+  - name: unit-test-long
+    tags: ["unit_test"]
```
Comment (lukech) [inline test/evergreen.yml]: Be cautious about setting this tag :) We only want this task to be executed under `ubuntu1804` build variant, for now.

File: test/evergreen.yml [hunk 2]
```diff
@@ -1027,6 +1027,16 @@ tasks:
...
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "unit test"
+        vars:
+          unit_test_args: ${unit_test_args} --long
```
Comment (lukech) [inline test/evergreen.yml]: Feel a bit over-engineered. `-v 2 --long` is fine to me.

File: test/evergreen.yml (hunk 1 above)
Comment (hbokhari) [reply test/evergreen.yml]: Ah yes I get your point. Will make the change.

File: test/evergreen.yml (hunk 2 above)
Comment (hbokhari) [reply test/evergreen.yml]: Noted.

**Comment by hbokhari** [comment]
Here is the link for patch build `with uni-test-long' : https://evergreen.mongodb.com/version/5dcca118d1fe070d30cc182e

**Pull request merged**